src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_ProhibitDTD.cs

```diff
@@ -208,7 +208,7 @@ public void v4()
         //[Variation(Desc = "v5.2- Test Default value of ProhibitDTD for Add(TextReader) for schema with DTD", Priority = 1, Params = new object[] { "bug356711_a.xsd", 0 })]
         [InlineData("bug356711_a.xsd", 0)]
         //[Variation(Desc = "v5.1- Test Default value of ProhibitDTD for Add(TextReader) with an xs:import for schema with DTD", Priority = 1, Params = new object[] { "bug356711.xsd", 0 })]
-        [InlineData("bug356711.xsd", 0)]
+        [InlineData("bug356711.xsd", 2)]
```

Comment:
where is this difference coming from?

Reply:
See https://github.com/dotnet/runtime/issues/30107